Sources/MarkupParser.swift

```diff
@@ -30,25 +28,37 @@ public struct MarkupParser {
 			case .leftDelimiter(let delimiter):
 				// Recursively parse all the tokens following the delimiter
 				openingDelimiters.append(delimiter)
-				elements.append(contentsOf: parse())
+				let childResult = parse()
+				if let childDelemiter = childResult.0, childDelemiter != delimiter && childResult.1.count == 1 {
```

Comment:
Fix the misspelling of `childDelemiter`. Should be `childDelimiter`

Reply:
done